api/external/applications/applications.proto

```diff
@@ -57,7 +57,15 @@ service ApplicationsService {
     option (chef.automate.api.policy).action = "list";
     option (chef.automate.api.iam.policy).resource = "applications:serviceGroups";
     option (chef.automate.api.iam.policy).action = "applications:serviceGroups:list";
-  }
+  };
+
+  rpc GetDisconnectedServices(DisconnectedServicesReq) returns (ServicesRes ) {
+    option (google.api.http).get = "/beta/applications/services/disconnected";
```

Comment:
Maybe a better route for this endpoint would be `/beta/applications/disconnected_services` so that when we work on the **delete disconnected services** endpoint, it will be called `/beta/applications/disconnected_services/delete` or just a `DELETE` request to this same endpoint. 🤔

Reply:
I would like to hear your thoughts @lancewf @srenatus 🥇  

**Thank you in advance!**

![tenor-18469822](https://user-images.githubusercontent.com/5712253/61034553-caee2400-a3c5-11e9-83c1-7d895ae46916.gif)

Reply:
> just a DELETE request to this same endpoint

I think that's preferrable. 😃